code/game/machinery/smelter.dm

```diff
@@ -128,6 +127,20 @@
 	for(var/obj/O in smelting.contents)
 		smelt_item(O)
 
+	//Occulus Edit Start: Smelting Mobs
+	for(var/mob/oopsalltori in smelting.contents)
```

Comment:
Fun police here -- I do think the in-joke is funny but future coders looking at this code might be confused because 'oopsalltori' and 'oopsallzack' aren't clear on a glance at what they refer to. This is a similar thing to how we have a variable called 'howdoitameahorseinminecraft' that I strongly dislike and intend to change because there's no clarity on what it does from a glance. I would ask you please change the variable names to something more descriptive (like `smelting_mob` and `external_organ`, so line 140 reads as `for(var/obj/item/organ/external/external_organ in smelting_mob.contents) ` or another similar descriptive variable)